content/applications/services/field_service/planning_itinerary.rst

```diff
@@ -18,10 +18,8 @@ Displaying your itinerary on a map
 ==================================
 
 .. important::
-   For a field service task to be featured on the map, it needs:
-
-   - a :guilabel:`Planned Date`;
-   - a valid address linked to the customer’s profile.
+   For a field service task to be featured on the map, the customer's profile needs to feature a
+   **valid address**.
```

Comment:
```suggestion
   For a field service task to be featured on the map, a **valid address* must be provided for the customer.
```
to avoid repeating "feature" twice :wink: